cleanlab/dataset.py

```diff
@@ -54,6 +54,26 @@ def rank_classes_by_label_quality(
 
     Only provide **exactly one of the above input options**, do not provide a combination.
 
+    Examples
+    --------
+    >>> from cleanlab.dataset import rank_classes_by_label_quality
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from sklearn.model_selection import cross_val_predict
+    >>> data, labels = get_data_labels_from_dataset(yourFavoriteDataset)
+    >>> yourFavoriteModel = LogisticRegression()
+    >>> num_crossval_folds = 3
```

Comment:
[nit] can just say cv=3 in the code below to save a line. We don't want these examples to take up too many lines, since then it is harder to scroll through the documentation